plugins/inputs/mongodb/mongodb_data.go

```diff
@@ -264,6 +265,27 @@ var ColDataStats = map[string]string{
 	"ok":               "Ok",
 }
 
+var TopDataStats = map[string]string{
```

Comment:
Please make this a private variable as it is only used within this package. If you want to improve the code, you might also want to change the variables above...

Reply:
done